setup.py

```diff
@@ -51,7 +51,8 @@
               ],
     'tensorflow': ["scikit-learn",
                    "tensorflow",
-                   "sklearn-crfsuite"
+                   "sklearn-crfsuite",
+                   "scipy"
```

Comment:
could you please move `tensorflow` below `scipy`, I think it is visually more clear

Reply:
Done so.